Source/Tools/FEXServer/Unwind/ELFMapping.cpp

```diff
@@ -426,4 +389,12 @@ namespace ELFMapping {
     return nullptr;
   }
 
+  ELFMemMapping::~ELFMemMapping() {
+    if (BaseRemapped) {
+      // If the FD was mapped then make sure to unmap it.
+      munmap(BaseRemapped, TotalSize);
```

Comment:
I think adding a destructor still leaves the implicit copy constructor around, so we should explicitly delete it to avoid accidental double-unmaps here.

Reply:
Removed the copy constructor.